src/app/shared/search/search-filters/search-filter/search-facet-filter-options/search-facet-option/search-facet-option.component.ts

```diff
@@ -119,4 +130,25 @@ export class SearchFacetOptionComponent implements OnInit {
     return getFacetValueForType(this.filterValue, this.filterConfig);
   }
 
+  /**
+   * Announces to the screen reader that the page will be reloaded, which filter has been selected
+   * and then implement “filterService.minimizeAll()”
+   */
+  selectingFilter() {
+    const filterValue = this.filterValue.value;
+    if (!this.selectedFilters.has(filterValue)) {
+      this.translateService.get('search-facet-option.update.announcement')
+        .subscribe(translatedMessage => {
+          const message = `${translatedMessage} ${filterValue}`;
+          this.liveRegionService.addMessage(message);
+
+          setTimeout(() => {
```

Comment:
I'm not sure I understand why you have to call `setTimeout` here.  It's already set by the `liveRegionService.addMessage()` call.   If there's a reason for this section of code, then I'd recommend describing it with a code comment.  Currently, it looks odd to me because I don't see any other areas in our current code where we call `setTimeout()` for a Live Region.